apps/studio/components/layouts/ProjectLayout/LayoutHeader/HelpPopover.tsx

```diff
@@ -86,8 +89,16 @@ const HelpPopover = () => {
             prioritized.
           </p>
           <div>
-            <Button asChild type="default" icon={<IconMail />}>
-              <Link href={supportUrl}>Contact Support</Link>
+            <Button
+              type="default"
+              icon={<HelpCircle size={15} strokeWidth={2.5} className="text-light" />}
+              onClick={() => {
+                setAiVariant('support')
+                setPages([COMMAND_ROUTES.AI])
+                setIsOpen(true)
+              }}
+            >
+              Contact Support
```

Comment:
The CTA language here probably needs changing - its weird that from a user's POV that i'm looking to contact support, but i end up with a modal that's chatting with a bot

Reply:
Just to understand the context of this PR - are we doing this to push users to self debug, and only reach out to support as a last resort? (i.e we're making it harder for users to find the support page?)

Reply:
I'm also just thinking from a user's POV, if something urgent happened to the project and they're frantically looking on how to reach support - this might frustrate them though, although this is (hopefully) not the common scenario

Reply:
@joshenlim It's an internal thing, we want to direct **Free** users to the “AI Chat” option before they can submit a ticket. Many of their issues can be resolved by the AI or our documentation and our goal is to reduce the number of tickets we receive each month and make the process more efficient for everyone. Also, the process is minimally invasive and they can easily skip it if they have an urgent problem. And the concept is not new, many other platforms have similar implementation so I believe ours should be well-received.